### ISSUE TITLE
Computing times using grnn_parpred

### ISSUE THREAD

**Issue by zepedropaulos** [statcompute/yager#3]
Hi,

I was testing your package with a sample dataset.
I'm using a training set/dataframe called `train` with roughly 52k observations and 5 variables, one being the target (52000x5). The test set, `test`, only contains 48 observations and the same 5 variables (48x5). 

Since I want to predict the test target value (48 observations) directly, I was not able to use the `grnn` package on its own, since it only assumes one observation at a time (at least I was not able to predict it as a set of 48 observations - matrix dimensions errors). Hence, I have 3 options:

1. Predict each one of the 48 observations with a for loop
2. Predict each one of the 48 observations with a for loop, but using paralelism via `%dopar%` of the `doParallel` package.
3. Predict the whole set using `grnn_parpred`, since it accepts the whole set as an input.

When comparing the processing times of each of the stated option, I get the following results:

1. For Loop = ~27 seconds
2. For Loop with doParallel = ~9 seconds
3. Using grnn_parpred = ~130 seconds

Using the regular for loop as a reference, the `doParallel` package solves the problem much much quickly than `grnn_parpred`, even by predicting each observation individually.

**Comment by statcompute** [comment]
First of all, thank you very much for testing my package. 

For only 48 records, I am not sure if it is necessary to generate predictions in parallel given the overhead. In addition, to ensure the compatibility, I used socket instead of folk in the parallel, which could be slower.
